src/rexsdesktop/controller/User.java

```diff
@@ -518,7 +518,7 @@ public static boolean verificarPin(String pin, String correo) {
 
     public static boolean actualizarPerfilUsuario(String nombreCompleto, String email, int id) {

```

Comment:
don't use a static method here, also why do you need this function returns a boolean?

Reply:
The boolean is to check if the data was updated correctly, I can make it void, if you suggest that.

Reply:
better throws a custom error if something fails